### ISSUE TITLE
Parameterization error when .rtf and .prm files are passed

### ISSUE THREAD

**Issue by GerardBCN** [Acellera/htmd#148]
When using newparameterize passing --rtf ff.rtf --prm ff.prm. These parameters were obtained from ParamChem. 

```
Traceback (most recent call last):
  File "/shared/gerard/anaconda3/bin/newparameterize", line 6, in <module>
    sys.exit(htmd.newparameterization.cli.main_parameterize())
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/cli.py", line 120, in main_parameterize
    mol = FFMolecule(filename=filename, method=method, netcharge=args.charge, rtf=args.rtf, prm=args.prm, basis=basis, execution=execution, qmcode=code )
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/ffmolecule.py", line 77, in __init__
    self._prm = PRM( prm )
  File "/shared/gerard/anaconda3/lib/python3.5/site-packages/htmd/newparameterization/ff.py", line 85, in __init__
    self.angles.append( AnglePrm([ ll[0], ll[1], ll[2] ], theta0=float(ll[4]), k0=float(ll[3]), rB=float(ll[6]), kB=float(ll[5]) ) )
TypeError: __init__() got an unexpected keyword argument 'kB'
```
[files.zip](https://github.com/Acellera/htmd/files/500941/files.zip)

**Comment by mj-harvey** [comment]
@GerardBCN mail me the problematic prm rtf, please

**Comment by GerardBCN** [comment]
They are attached to my issue. They come from ParamChem

**Comment by mj-harvey** [comment]
Fixed in 7478dc3

**Comment by j3mdamas** [comment]
@mj-harvey can we close?
